### PR TITLE
Add Zulip

### PR DESCRIPTION
**Pull request by jayvdb**
Has at least Errbot and Hubot connectors, both of which are used by several orgs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -306,6 +306,11 @@ These open source projects will help you to build a bot in order to automate you
 
 * [Hubot](https://hubot.github.com/) [adapter](https://github.com/athieriot/hubot-yammer) (CoffeeScript, Node.js)
 
+### Zulip
+
+* [Errbot](http://errbot.io/) [adapter](https://github.com/zulip/errbot-backend-zulip) (Python)
```
Comment (exAspArk) [inline README.md:311]: @jayvdb wow, awesome!

Could you please add `+` (forgot to add it for `Yammer`)? 🙏
So, it'll look like:

```
* Errbot + adapter (Python)
* Hubot + adapter (CoffeeScript, Node.js)
```

Comment (jayvdb) [reply README.md:311]: :/  https://github.com/exAspArk/awesome-chatops/blob/master/CONTRIBUTING.md#for-bots-section says not to add the `+` , so I guess that needs fixing?

Comment (exAspArk) [reply README.md:311]: Yep, right 💯 
I'll update it

**Comment by exAspArk** [comment]
Thanks for contributing!

**Pull request merged**